cosmos.config.json

```diff
@@ -5,5 +5,6 @@
   "webpack": {
     "configPath": "react-scripts/config/webpack.config",
     "overridePath": "cosmos.override.js"
-  }
+  },
+  "port": 5001
```

Comment:
revert

Reply:
I'm adding this to help out people like @willhennessy who have issues w/ port 5000. I just got a new computer and ran into it too.